### ISSUE TITLE
It kills css import src

### ISSUE THREAD

**Issue by saturday1** [SoapBox/linkifyjs#78]
Hello! I tried to implement this into our site. The problem (I know this is not semantically right) is that we have style elements inside the body element, with import. We have to use this solution within our solution in some cases.

This is what the script do with a file import:
```
<style media="screen" type="text/css"> @import url("<a class="linkified" target="_blank" cdn.axiell.com="" arena="">//cdn.axiell.com/arena/instagram-feed.css")</a>; </style>
```

As you can see it pushes the a-tag inside the import which kind of kills the look of different parts on the page.

In other words, this solution does not work:

```html
<div class="arena-columns-1-2-1-2-1">
Linkify this link: www.i-should-be-linkyfied.com

But this is HTML and should not be linkyfied:
<style media="screen" type="text/css">
@import url("//cdn.axiell.com/arena/instagram-feed.css");
</style>
</div>
```
```javascript
<script src="//cdn.axiell.com/arena/linkify.min.js"></script>
<script src="//cdn.axiell.com/arena/linkify-jquery.min.js"></script>
<script>
jQuery(document).ready(function(){
	jQuery('.arena-columns-1-2-1-2-1').linkify({
		target: "_blank"
	});
});
</script>
```

Anyone know how to solve? :)

**Comment by saturday1** [comment]
I thought I figured out a solution, so I tried this. The script seems right (The console.log shows that there is only one style element in the HTML above). But still it linkyfies the import src anyway, like it runs after the each and if statement:

```javascript
jQuery(document).ready(function(){
	// LOOP THROUGH EVERY ELEMENT IN WITHIN BODY
	jQuery('body *').each(function(){
		// IF THE ELEMENT IS NOT(!) STYLE OR SOME SOME SPECIFIC IDS, LINKYFY IT
		if(!jQuery(this).is('style, #instagram-feed, #twitter-feed, #facebook-feed')){
			jQuery(this).linkify();
			
			// JUST FOR TESTING
			console.log("Not style element");
			
			// JUST FOR TESTING
		} else {
			console.log("style element");
		};
	});
});
```

**Comment by nfrasser** [comment]
This has been partially addressed in #106 with the new `validate` option. #57 should also be fixed before this is closed.
